config/initializers/sidekiq.rb

```diff
@@ -2,9 +2,6 @@
 
 require 'three_scale/sidekiq_retry_support'
 require 'three_scale/sidekiq_logging_middleware'
-require 'sidekiq/throttled'
-
-Sidekiq::Throttled.setup!
```

Comment:
Why don't we need this anymore?...

Reply:
Thanks for the explanation @jlledom

Reply:
But in the [readme](https://github.com/ixti/sidekiq-throttled/tree/main) I still see `require "sidekiq/throttled"` is needed :thinking:

Reply:
@akostadinov We don't need to `require 'sidekiq/throttled'` because it's already being required from:

https://github.com/3scale/porta/blob/89a91607b8387892967a0b9315cca9e0d36c99e8/config/application.rb#L38